python/mxnet/gluon/block.py

```diff
@@ -1026,6 +1030,53 @@ def _call_cached_op(self, *args):
             out = [out]
         return _regroup(out, self._out_format)
 
+    def optimize_for(self, x, *args, backend=None, backend_opts=None, **kwargs):
```

Comment:
is this also missing auxs ?

Reply:
nope. in Gluon args here means inputs to the model. not the same as args/aux in symbol/module flow